chromium_src/chrome/browser/ui/webui/settings/people_handler.cc

```diff
@@ -3,13 +3,22 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this file,
  * You can obtain one at http://mozilla.org/MPL/2.0/. */
 
+#include "brave/components/sync/driver/brave_sync_service_impl.h"
+
 // IsSetupInProgress isn't accurate in brave sync flow especially for the first
 // time setup, we rely on it to display setup dialog
-#define BRAVE_GET_SYNC_STATUS_DICTIONARY  \
-  sync_status.SetBoolPath(                \
-      "firstSetupInProgress",             \
-      service && !disallowed_by_policy && \
-          !service->GetUserSettings()->IsFirstSetupComplete());
+#define BRAVE_GET_SYNC_STATUS_DICTIONARY                                    \
+  sync_status.SetBoolPath(                                                  \
+      "firstSetupInProgress",                                               \
+      service && !disallowed_by_policy &&                                   \
+          !service->GetUserSettings()->IsFirstSetupComplete());             \
+  {                                                                         \
+    syncer::BraveSyncServiceImpl* brave_sync_service =                      \
+        static_cast<syncer::BraveSyncServiceImpl*>(service);                \
+    bool failed_to_decrypt = false;                                         \
+    brave_sync_service->prefs().GetSeed(&failed_to_decrypt);                \
+    sync_status.SetBoolPath("syncWordsDecryptionError", failed_to_decrypt); \
```

Comment:
nit: since this is a `bool` then perhaps better variable name would be `hasSyncWordsDecryptionError`

Reply:
good catch, thanks, fixed at https://github.com/brave/brave-core/compare/9f94be62f077007af6b124c089e3dfc5b6efd6b7..980e5dc37026a0bd8bb1e898e35f88e51ce1f5c2#diff-6314ab9f5910c0e3dd53f8ac6bf09b584020abe6790abe7ebcdacc5756c5d5f9R315